internal/handler/msg_find.go

```diff
@@ -145,6 +145,10 @@ func (h *Handler) MsgFind(connCtx context.Context, msg *wire.OpMsg) (*wire.OpMsg
 		t = cursor.TailableAwait
 	}
 
+	if params.NoCursorTimeout {
+		t = cursor.NoTimeout
```

Comment:
Does GitHub Copilot Workspaces allow you to compile code? This identifier is not defined, so the code does not compile.

Reply:
It does not. I'm sorry for the noise. I should have created a draft PR. I was trying Workspaces and clicked to create a PR.